src/app/shared/interfaces/feature-info.interface.ts

```diff
@@ -2,10 +2,11 @@ import {GeometryWithSrs} from './geojson-types-with-srs.interface';
 import {HasMetaDataLink} from './metaDataLink.interface';
 import {IsSingleLayer} from './single-layer.interface';
 import {LinkObject} from './link-object.interface';
+import {Image} from '../models/gb3-api-generated.interfaces';
```

Comment:
I add this comment here, but it also holds true for others: With this, your essentially coupling all internal components and interfaces to the API. If they change it, our interfaces and our components will break.

Generated interfaces should be abstracted and only used within the services, and then mapped.

Reply:
I created a new interface which copies the generated Image interace (analog to LinkObject) and replaced all the imports